### PR TITLE
Logging 4.5 Documentation Issues

### PR DESCRIPTION
**Pull request by mburke5678**
Peri comments: https://docs.google.com/document/d/1MT6jiT8wr56FkCAX2qYAZryDpGv-ysqkp-ApKg_htzw/edit?disco=AAAAGkOngQs
Jeff C comments: https://docs.google.com/document/d/11v0exwdxLXf3UECs6GatKwH5j4aRaR8ZoxnSKBYvFmM/edit?ts=5ee7ca0e#

### CODE REVIEW

File: modules/cluster-logging-configuring-image-about.adoc [hunk 1]
```diff
@@ -16,17 +16,18 @@ $ oc -n openshift-logging set env deployment/cluster-logging-operator --list | g
...
+PROMTAIL_IMAGE=quay.io/openshift/origin-promtail:latest <3>
 ----
-<1> *ELASTICSEARCH_IMAGE* deploys Elasticsearch.
-<2> *FLUENTD_IMAGE* deploys Fluentd.
-<3> *KIBANA_IMAGE* deploys Kibana.
-<4> *CURATOR_IMAGE* deploys Curator.
-<5> *OAUTH_PROXY_IMAGE* defines OAUTH for {product-title}.
+<1> *FLUENTD_IMAGE* deploys Fluentd.
+<2> *CURATOR_IMAGE* deploys Curator.
+<3> *PROMTAIL_IMAGE* defines OAUTH for {product-title}.
```
Comment (periklis) [inline modules/cluster-logging-configuring-image-about.adoc:25]: Drop PROMTAIL_IMAGE. We don't officialy support it.

**Comment by mburke5678** [comment]
NOTE TO SELF: Update file:///home/mburke/Docs/openshift-docs/_preview/openshift-enterprise/logging-efk-remove/logging/cluster-logging.html#cluster-logging-about-logstore_cluster-logging

File: modules/cluster-logging-collector-pod-location.adoc [hunk 2]
```diff
@@ -14,13 +14,12 @@ Run the following command in the `openshift-logging` project:
 ----
 $ oc get pods --all-namespaces -o wide | grep fluentd
```
Comment (anpingli) [inline modules/cluster-logging-collector-pod-location.adoc:15]: How about use the --selector? 
$oc get pods --selector component=fluentd -o wide -n openshift-logging
NAME            READY   STATUS    RESTARTS   AGE    IP            NODE                  NOMINATED NODE   READINESS GATES
fluentd-8d69v   1/1     Running   0          134m   10.130.2.30   master1.example.com    <none>           <none>
fluentd-bd225   1/1     Running   0          134m   10.131.1.11   master2.example.com    <none>           <none>
fluentd-cvrzs   1/1     Running   0          134m   10.130.0.21   master3.example.com    <none>           <none>
fluentd-gpqg2   1/1     Running   0          134m   10.128.2.27   worker1.example.com    <none>           <none>
fluentd-l9j7j   1/1     Running   0          134m   10.129.2.31   worker2.example.com    <none>           <none>

File: modules/cluster-logging-configuring-image-about.adoc [hunk 3]
```diff
@@ -16,17 +16,16 @@ $ oc -n openshift-logging set env deployment/cluster-logging-operator --list | g
 ----
 
 ----
-ELASTICSEARCH_IMAGE=registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.3 <1>
-FLUENTD_IMAGE=registry.redhat.io/openshift4/ose-logging-fluentd:v4.3 <2>
-KIBANA_IMAGE=registry.redhat.io/openshift4/ose-logging-kibana5:v4.3 <3>
-CURATOR_IMAGE=registry.redhat.io/openshift4/ose-logging-curator5:v4.3 <4>
-OAUTH_PROXY_IMAGE=registry.redhat.io/openshift4/ose-oauth-proxy:v4.3 <5>
+FLUENTD_IMAGE=quay.io/openshift-qe-optional-operators/ose-logging-fluentd@sha256:ffdf79da7386871d2bc24cd937e02284b30f85a9979dc8c635fb73021cbca2f3 <1>
```
Comment (anpingli) [inline modules/cluster-logging-configuring-image-about.adoc:19]: Please replace quay.io/openshift-qe-optional-operators with registry.redhat.io/openshift4/.

File: modules/cluster-logging-deploying-about.adoc [hunk 4]
```diff
@@ -161,14 +161,16 @@ spec:
...
         limits:
-          memory: 2Gi
+          memory: 16Gi
         requests:
           cpu: 200m
-          memory: 2Gi
-      storage: {}
+          memory: 16Gi
+        storage:
+          storageClassName: "gp2"
```
Comment (anpingli) [inline modules/cluster-logging-deploying-about.adoc:172]: To align with another PR, shall we replace gp2?

File: modules/cluster-logging-deploying-about.adoc [hunk 5]
```diff
@@ -161,14 +161,16 @@ spec:
   logStore:
     type: "elasticsearch"
     elasticsearch:
-      nodeCount: 2
+      nodeCount: 3
```
Comment (anpingli) [inline modules/cluster-logging-deploying-about.adoc:164]: Could we add  retentionPolicy here?
      retentionPolicy: <1>
        application:
          maxAge: 1d
        infra:
          maxAge: 7d
        audit:
          maxAge: 7d

File: modules/cluster-logging-elasticsearch-status-comp.adoc [hunk 6]
```diff
@@ -78,7 +82,7 @@ Containers:
   proxy:
     Container ID:  cri-o://3f77032abaddbb1652c116278652908dc01860320b8a4e741d06894b2f8f9aa1
     State:          Running
-      Started:      Mon, 08 Apr 2019 10:18:38 -0400
+      Started:      Mon, 08 Jun 2020 10:18:38 -0400
```
Comment (anpingli) [inline modules/cluster-logging-elasticsearch-status-comp.adoc:85]: registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.3   ->registry.redhat.io/openshift4/ose-logging-elasticsearch6@sha256:4265742c7cdd85359140e2d7d703e4311b6497eec7676957f455d6908e7b1c25

By the way, it is better to update digest after we kick-off the 4.5.

File: modules/cluster-logging-eventrouter-deploy.adoc [hunk 7]
```diff
@@ -154,17 +154,17 @@ configmap/logging-eventrouter created
 deployment.apps/logging-eventrouter created
 ----
 
-. Validate that the Event Router installed:
+. Validate that the Event Router installed in the `openshift-logging` project:
```
Comment (anpingli) [inline modules/cluster-logging-eventrouter-deploy.adoc:157]: Will we drop Event router in 4.5?

File: modules/cluster-logging-deploying-about.adoc (hunk 5 above)
Comment (mburke5678) [reply modules/cluster-logging-deploying-about.adoc:164]: Will be added by https://github.com/openshift/openshift-docs/pull/22403/files#diff-2230f32010df8cbe4fb588517ca026f7

File: modules/cluster-logging-deploying-about.adoc (hunk 4 above)
Comment (mburke5678) [reply modules/cluster-logging-deploying-about.adoc:172]: This CR is supposed to show what a modified CR looks like. I think having a specific storageClassName is OK here.

File: modules/cluster-logging-elasticsearch-status-comp.adoc (hunk 6 above)
Comment (mburke5678) [reply modules/cluster-logging-elasticsearch-status-comp.adoc:85]: Outdated

File: modules/cluster-logging-eventrouter-deploy.adoc (hunk 7 above)
Comment (mburke5678) [reply modules/cluster-logging-eventrouter-deploy.adoc:157]: Yes. I just removed the two event router files. I removed event router from the TOC earlier.

**Review by anpingli — COMMENTED** [review]
LGTM

**Comment by mburke5678** [comment]
Minor editorial changes. No peer review needed.

**Pull request merged**